### PR TITLE
Friend Link Component

### PR DESCRIPTION
**Pull request by GuyInFridge**

### CODE REVIEW

File: src/app/signing/signing.component.css [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+	height: 10em;
+	resize: none; /*both;*/
+	margin-top: 2em;
+	margin-left:auto;
+	margin-right:auto;
+}
+
+#options > * {
+	margin: 2em 0.5em;
+}
```
Comment (TommyHuang-dev) [inline src/app/signing/signing.component.css:13]: Signing component stuff is in friend-link. Please make sure there are no merge conflicts!

File: src/app/signing/signing.component.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
+<div class="container py-5 center-children" style="margin: 25px auto">
```
Comment (TommyHuang-dev) [inline src/app/signing/signing.component.html:1]: Same as above... This also looks like an old version of signing component so try not to override the newer changes.

File: src/app/friend-link/friend-link.component.css [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+    margin: 0 1em;
+    margin-top: 2em;
+    font-size: 1.1em;
+}
+
+#title-text {
+    /* We see you’re trying to sign your friend’s virtrolio... */
+    text-align: center;
+    font-size: 2em;
+}
```
Comment (ricetech) [inline src/app/friend-link/friend-link.component.css:17]: No newline at end of file
```suggestion
}

```

File: src/app/signing/signing.component.css (hunk 1 above)
Comment (ricetech) [inline src/app/signing/signing.component.css:13]: No newline at end of file
```suggestion
}

```

**Pull request merged**